src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -290,6 +290,12 @@ def load_arguments(self, _):
         c.argument('node_resource_group')
         c.argument('enable_defender', action='store_true')
         c.argument('defender_config', validator=validate_defender_config_parameter)
+        c.argument('enable_disk_driver', action='store_true')
+        c.argument('disable_disk_driver', action='store_true')
+        c.argument('enable_file_driver', action='store_true')
+        c.argument('disable_file_driver', action='store_true')
+        c.argument('enable_snapshot_controller', action='store_true')
+        c.argument('disable_snapshot_controller', action='store_true')
```

Comment:
This is under the scope of `aks create`, should declare disable_xxx only and missing enable_xx & disable_xxx for `aks update`

Reply:
done